CMakeLists.txt

```diff
@@ -0,0 +1,284 @@
+     endif ()
+     find_package(AMReX CONFIG REQUIRED ${AMREX_REQUIRED_COMPONENTS} )
+  endif ()
+
+endif ()
+
+set(SUNDIALS_HOME "" CACHE PATH "Path to SUNDIALS top-level source directory")
+
+if (SUNDIALS_HOME)  # SUPERBUILD MODE
+  message(STATUS "SUNDIALS top-level source directory path: ${SUNDIALS_HOME}")
+
+  # Add SUNDIALS module path
+  list(APPEND CMAKE_MODULE_PATH ${SUNDIALS_HOME}/cmake)
+  set(SUNDIALS_BINARY_DIR "${CMAKE_BINARY_DIR}/sundials" CACHE INTERNAL "Path to SUNDIALS top-level binary directory")
+
+  # Set required settings for SUNDIALS
+  set(SUNDIALS_PRECISION   ${PELEPHYSICS_PRECISION} CACHE STRING "Floating point precision" FORCE)
+  set(BUILD_ARKODE         ON                       CACHE INTERNAL "")
+  set(BUILD_CVODE          ON                       CACHE INTERNAL "")
+  set(SUNDIALS_INDEX_SIZE  32                       CACHE INTERNAL "")
+  set(BUILD_CVODES         OFF                      CACHE INTERNAL "")
+  set(BUILD_IDA            OFF                      CACHE INTERNAL "")
+  set(BUILD_IDAS           OFF                      CACHE INTERNAL "")
+  set(BUILD_KINSOL         OFF                      CACHE INTERNAL "")
+  set(BUILD_CPODES         OFF                      CACHE INTERNAL "")
+  set(BUILD_EXAMPLES       OFF                      CACHE INTERNAL "")
+  set(_BUILD_EXAMPLES      OFF                      CACHE INTERNAL "")
+  set(BUILD_TESTING        OFF                      CACHE INTERNAL "")
+  set(EXAMPLES_ENABLE_C    OFF                      CACHE INTERNAL "")
+  set(EXAMPLES_ENABLE_CXX  OFF                      CACHE INTERNAL "")
+  set(EXAMPLES_INSTALL     OFF                      CACHE INTERNAL "")
+  set(ENABLE_CUDA          ${PELEPHYSICS_CUDA}      CACHE INTERNAL "")
+  set(ENABLE_HIP           ${PELEPHYSICS_HIP}       CACHE INTERNAL "")
+  set(ENABLE_SYCL          ${PELEPHYSICS_SYCL}      CACHE INTERNAL "")
+  set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+
+  add_subdirectory(${SUNDIALS_HOME} ${SUNDIALS_BINARY_DIR})
+
+  message(STATUS "SUNDIALS binary directory: ${SUNDIALS_BINARY_DIR}")
+else ()
+
+#
+# Find SUNDIALS, check for required components
+#
+  if (NOT TARGET SUNDIALS::cvode)
+    # Check if we can require components
+    find_package(SUNDIALS)
+  endif ()
+
+endif ()
+
+#
+# Enable CUDA if requested
+#
+if (PELEPHYSICS_GPU_BACKEND STREQUAL "CUDA")
+   # CMake 3.18+: CMAKE_CUDA_ARCHITECTURES
+   # https://cmake.org/cmake/help/latest/policy/CMP0104.html
+   if(POLICY CMP0104)
+      cmake_policy(SET CMP0104 OLD)
+   endif()
+   include(AMReX_SetupCUDA)
+   include(AMReXTargetHelpers)
+endif ()
+
+
+#
+# Define the object library to compile
+#
+set(pelephysics_lib_name PelePhysics-${PELEPHYSICS_CHEMISTRY_MODEL}-${PELEPHYSICS_EOS_MODEL}-${PELEPHYSICS_TRANSPORT_MODEL})
+add_library(${pelephysics_lib_name} OBJECT)
+if (BUILD_SHARED_LIBS)
+  set_target_properties(${pelephysics_lib_name} PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

Comment:
I never really know which way to go with this. Sometimes it's good to always have PIC on, but I think it's best to start by letting CMake handle it by removing this.